### PR TITLE
Update apis

### PR DESCRIPTION
**Pull request by BeshoyMelika**

### CODE REVIEW

File: example/lib/apis/user_apis.dart [hunk 1]
```diff
@@ -5,6 +5,11 @@ callUpdateProfileApi(String name, String mobile) {
       name, mobile, (bool isUpdated) {}, (ApiErrorModel error) {});
 }
 
+updateNotificationSettingApi(bool notification) {
```
Comment (MinaSamirCap) [inline example/lib/apis/user_apis.dart]: can we rename the notification variable to shouldEnableNotification

File: lib/graphql/fragments.graphql [hunk 2]
```diff
@@ -40,6 +42,7 @@ fragment User on  User{
     name
     email
     mobile
+    is_notifiable
```
Comment (MinaSamirCap) [inline lib/graphql/fragments.graphql:45]: why we are in need to add is_notifiable?

File: lib/apis/user_api_manager.dart [hunk 3]
```diff
@@ -32,6 +32,21 @@ class UserApiManager extends BaseApiManager {
     }
   }
 
+  static Future<void> updateNotificationSettingApi(
+      bool notificationSetting, Function success, Function fail) async {
```
Comment (MinaSamirCap) [inline lib/apis/user_api_manager.dart]: rename notificationSetting to shouldEnableNotification

**Review by MinaSamirCap — CHANGES_REQUESTED** [review]

File: lib/graphql/fragments.graphql (hunk 2 above)
Comment (BeshoyMelika) [reply lib/graphql/fragments.graphql:45]: to know notification for this user on or off

File: lib/apis/user_api_manager.dart (hunk 3 above)
Comment (BeshoyMelika) [reply lib/apis/user_api_manager.dart]: done

File: example/lib/apis/user_apis.dart (hunk 1 above)
Comment (BeshoyMelika) [reply example/lib/apis/user_apis.dart]: done

File: example/lib/apis/user_apis.dart [hunk 4]
```diff
@@ -5,6 +5,11 @@ callUpdateProfileApi(String name, String mobile) {
       name, mobile, (bool isUpdated) {}, (ApiErrorModel error) {});
 }
 
+shouldEnableNotification(bool notification) {
```
Comment (MinaSamirCap) [inline example/lib/apis/user_apis.dart]: change the signature to
enableNotification(bool shouldEnableNotification) {
  UserApiManager.shouldEnableNotification(
      shouldEnableNotification, (bool isEnabled) {}, (ApiErrorModel error) {});
}

File: example/lib/main.dart [hunk 5]
```diff
@@ -136,6 +136,16 @@ class _MyHomePageState extends State<MyHomePage> {
               ),
             ],
           ),
+          RaisedButton(
+            color: Colors.brown,
+            onPressed: () {
+              shouldEnableNotification(false);
```
Comment (MinaSamirCap) [inline example/lib/main.dart]: change the name according to the new requests

File: lib/apis/user_api_manager.dart (hunk 3 above)
Comment (MinaSamirCap) [reply lib/apis/user_api_manager.dart]: follow this signature
static Future<void> updateNotificationSetting(
      bool shouldEnableNotification, Function success, Function fail) async {

**Review by MinaSamirCap — CHANGES_REQUESTED** [review]

File: example/lib/apis/user_apis.dart (hunk 4 above)
Comment (BeshoyMelika) [reply example/lib/apis/user_apis.dart]: done

File: lib/apis/user_api_manager.dart (hunk 3 above)
Comment (BeshoyMelika) [reply lib/apis/user_api_manager.dart]: done

**Review by MinaSamirCap — APPROVED** [review]

**Pull request merged**